src/main/presenter/agentPresenter/acp/agentToolManager.ts

```
@@ -773,12 +977,25 @@ export class AgentToolManager {
+          )
+          return { content: await fileSystemHandler.editFile(parsedArgs, baseDirectory) }
         case 'execute_command':
           if (!this.bashHandler) {
             throw new Error('Bash handler not initialized for execute_command tool')
           }
+          const commandResult = await this.bashHandler.executeCommand(parsedArgs, {
+            conversationId
+          })
           return {
-            content: await this.bashHandler.executeCommand(parsedArgs, { conversationId })
+            content:
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)